code42_connector.py

```diff
@@ -65,19 +74,45 @@ def _handle_test_connectivity(self, param):
             # Ideally the client instantiation would happen in `initialize()` but that function does not have access
             # to the action, so it cannot effectively report errors to the UI.
             # Fix this with a decorator or something similar.
-            self._client = py42.sdk.from_local_account(self._cloud_instance, self._username, self._password)
-            self._client.users.get_current()
+            self.client.users.get_current()
         except Exception as exception:
             return action_result.set_status(phantom.APP_ERROR, f"Unable to connect to Code42: {str(exception)}")
 
         self.save_progress("Test Connectivity Passed")
         return action_result.set_status(phantom.APP_SUCCESS)
 
+    def _handle_add_departing_employee(self, param):
```

Comment:
Should we handle specific exceptions where that we can give nice error messages, like when you try to add a user that's already added, user doesn't exist, etc?

Reply:
I thought we would have to, but the nice error messages from py42 already custom enough imo when they happen in the UI

Reply:
e.g. 

```
handle_action exception occurred. Error string: 'User with ID 921286907298179098 is already on the departing-employee list.'
```

Reply:
nice!